bciers/libs/components/src/form/SingleStepTaskListForm.tsx

```diff
@@ -40,18 +40,12 @@ const SingleStepTaskListForm = ({
 }: SingleStepTaskListFormProps) => {
   const hasFormData = Object.keys(rawFormData).length > 0;
   const formData = hasFormData ? createNestedFormData(rawFormData, schema) : {};
+  const [formState, setFormState] = useState(formData);
```

Comment:
Just for clarity since I cleaned up all that commented code, this is the actual fix.